crates/polkavm/src/tests.rs

```diff
@@ -517,6 +517,88 @@ impl TestInstance {
     }
 }
 
+fn blob_null_pointer_djump() -> ProgramBlob {
+    let mut builder = ProgramBlobBuilder::new();
+    builder.set_rw_data_size(0x4000);
+    builder.add_export_by_basic_block(0, b"main");
+    builder.set_code(
+        &[
+            asm::jump_indirect(Reg::A0, 0x0),
+        ],
+        &[0x0],
```

Comment:
```suggestion
        &[],
```

First entry of the jump table is already implicitly zero.